From: Martin T. H. Sandsmark <martin.sandsmark@kde.org>
Date: Tue, 07 Aug 2012 03:35:54 +0000
Subject: We need to consider the possibility of an Attention wanted flag.
X-Git-Url: http://quickgit.kde.org/?p=kde-workspace.git&amp;a=commitdiff&amp;h=c4e93e26e400ff84a9f0a6cd3fd353dbd382c3db
---
We need to consider the possibility of an Attention wanted flag.
---


--- a/plasma/desktop/applets/tasks/abstracttaskitem.cpp
+++ b/plasma/desktop/applets/tasks/abstracttaskitem.cpp
@@ -231,7 +231,9 @@
     m_flags = flags;
 
     QString newBackground;
-    if (m_flags & TaskIsMinimized) {
+    if (m_flags & TaskWantsAttention) {
+        newBackground = "attention";
+    } else if (m_flags & TaskIsMinimized) {
         newBackground = "minimized";
     } else if (m_flags & TaskHasFocus) {
         newBackground = "focus";

